### ISSUE TITLE
Unable to connect to Parallel Service

### ISSUE THREAD

**Issue by aanfarhan** [utsanjan/PD-Runner#28]
<img width="306" alt="image" src="https://user-images.githubusercontent.com/19621476/191697216-148a70d4-cdf9-450b-a5cd-609f144bc004.png">
Hi, i am using latest PD Runner 3.8 patched and followed the instruction on README. But I am getting this errors, my Parallel version is 17.1.2 and Using M1 with Monterey 12.6

**Comment by utsanjan** [comment]
Update your Parallels Desktop. It only works with PD 18 and above.

**Comment by aanfarhan** [comment]
I just upgrade my PD to 18 and when start the VM it says "Failed to start make sure you are already trial user..." Do I need to change my account ? I am already a trial user when using PD 17

**Comment by utsanjan** [comment]
No, after upgrading you have to use the patch command once more.
